src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -305,6 +371,10 @@ def update_existing_check_in(
         )
 
     if not valid_duration(updated_duration):
+        processing_errors.append(
+            ProcessingError(ProcessingErrorType.CHECKIN_INVALID_DURATION),
+            {"duration": updated_duration},
+        )
```

Comment:
is this correct? shouldn't the `{"duration": updated_duration}` be in the `ProcessingError`

Reply:
Oops yeah, good catch